src/main/java/com/leveluptasks/controller/UserController.java

```diff
@@ -61,6 +62,12 @@ public ResponseEntity<User> login(@RequestBody UserDto userDto) throws NoSuchAlg
         }
     }
 
+    @Operation(summary = "Find user with email", description = "Find user with email")
+    @PostMapping("/email/{email}")
```

Comment:
mmmh it is not a bad practice but whyu don't you parameter this route as a getmapping, to avoid vulnerabilities ?